tgui/packages/tgui/interfaces/Electropack.js

```diff
@@ -0,0 +1,78 @@
+import { toFixed } from 'common/math';
+import { useBackend } from '../backend';
+import { Button, LabeledList, NumberInput, Section } from '../components';
+import { Window } from '../layouts';
+
+export const Electropack = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    power,
+    code,
+    frequency,
+    minFrequency,
+    maxFrequency,
+  } = data;
+  return (
+    <Window>
+      <Window.Content>
+        <Section>
+          <LabeledList>
+            <LabeledList.Item label="Power">
+              <Button
+                icon={power ? 'power-off' : 'times'}
+                content={power ? 'On' : 'Off'}
+                selected={power}
+                onClick={() => act('power')} />
+            </LabeledList.Item>
+            <LabeledList.Item
+              label="Frequency"
+              buttons={(
+                <Button
+                  icon="sync"
+                  content="Reset"
+                  onClick={() => act('reset', {
+                    reset: "freq",
+                  })} />
+              )}>
+              <NumberInput
+                animate
+                unit="kHz"
+                step={0.2}
+                stepPixelSize={6}
+                minValue={minFrequency / 10}
+                maxValue={maxFrequency / 10}
+                value={frequency / 10}
+                format={value => toFixed(value, 1)}
+                width="80px"
+                onDrag={(e, value) => act('freq', {
```

Comment:
Use `onChange` here so it only sends the topic to DD when the user lets go, instead of on each increment.